packages/ui/tests/e2e/pages/navigation.base.spec.ts

```diff
@@ -69,3 +69,87 @@ test('Navigation - Locks', async ({page}) => {
     await expect(page).toHaveTitle(makeClusterTille({page: 'Navigation', path: 'locked'}));
     await expect(page).toHaveURL(makeClusterUrl(`navigation?navmode=locks&path=${E2E_DIR}/locked`));
 });
+
+test('Navigation - URL correct encoding', async ({page}) => {
+    test.slow();
+    await test.step('escaped-symbol', async () => {
+        await page.goto(makeClusterUrl(`navigation?navmode=content&path=${E2E_DIR}/bad-names`));
+        const link = await page.waitForSelector(
+            `[href="/${CLUSTER}/navigation?path=${E2E_DIR}/bad-names/escaped-symbol%0A"]`,
```

Comment:
Is it possible to add corresponding `:text("...")` pseudo selectors for all nodes from bad names?